lib/services/apiAppManagement/package.json

```diff
@@ -17,7 +17,7 @@
     "management",
     "apiapp"
   ],
-  "main": "./lib/apiapp.js",
+  "main": "./lib/apiApp.js",
```

Comment:
Yeah, it turns out to be really hard to rename a file with just a case difference on a windows box. :-)